### PR TITLE
Enable single class coco subsampling

### PR DESCRIPTION
**Pull request by ivan-lazarevich**

### CODE REVIEW

**Comment by olivier-dl** [comment]
Hey @ivan-lazarevich, since Yasser is swamped with stuff he asked me to review this one. Can you edit the description to provide details as to why this PR is being done and what it is achieving? So I can get up to speed and understand and review correctly. Thanks :)

File: deeplite_torch_zoo/src/objectdetection/eval/coco/coco_evaluator.py [hunk 1]
```diff
@@ -32,7 +34,7 @@ def __init__(
 
     def evaluate(self, multi_test=False, flip_test=False):
         results = []
-        for img, _, _, img_ind in self.dataset:
+        for img, _, _, img_ind in tqdm(self.dataset):
```
Comment (ahmed-deeplite) [inline deeplite_torch_zoo/src/objectdetection/eval/coco/coco_evaluator.py:37]: Ehsan doesn't like these in evaluators. They make the output too noisy when you run neutrino... Lets avoid tqdm in evaluators

**Review by ahmed-deeplite — APPROVED** [review]
One minor comment but all is good. Thanks Ivan!

File: deeplite_torch_zoo/src/objectdetection/eval/coco/coco_evaluator.py (hunk 1 above)
Comment (ivan-lazarevich) [reply deeplite_torch_zoo/src/objectdetection/eval/coco/coco_evaluator.py]: OK, I disabled tqdm by default, but kept it for torch zoo training :Ь

**Pull request merged**